packages/vuetify/src/components/VSlideGroup/VSlideGroup.tsx

```diff
@@ -108,6 +108,21 @@ export const VSlideGroup = genericComponent<VSlideGroupSlots>()({
       return group.items.value.findIndex(item => item.id === group.selected.value[group.selected.value.length - 1])
     })
 
+    const items = computed(() => {
+      const items = []
+      const children = contentRef.value?.children
+      if (children) {
+        for (const child of children) {
+          const style = getComputedStyle(child)
+          items.push(isHorizontal.value
+            ? child.clientWidth + parseInt(style.marginLeft) + parseInt(style.marginRight)
+            : child.clientHeight + parseInt(style.marginTop) + parseInt(style.marginBottom)
```

Comment:
DOM access isn't reactive, this will only run once.

Reply:
@KaelWD - I was just thinking about change in isHorizontal value
But I get your point that this will not work in case of dynamic addition of VSlideGroupItem